src/SprykerEco/Shared/AfterPay/Transfer/after-pay.transfer.xml

```diff
@@ -202,9 +202,9 @@
     <transfer name="AfterPayAvailablePaymentMethodsResponse">
         <property name="checkoutId" type="string"/>
         <property name="outcome" type="string"/>
-        <property name="customer" type="array" singular="customer"/>
+        <property name="customers" type="array" singular="customer"/>
```

Comment:
You can not change existing ones, this would be a heavy BC break!

Reply:
Without this changes, Spryker CI build is failed. Because define singular is nessesary for checking.
So, We need rename **customer** to **customers** and and add singular field.
Also, I found place where this field is used:
`Zed/AfterPay/Business/Api/Adapter/ApiCall/AvailablePaymentMethodsCall.php line 87`

and I changed calling this function

Reply:
No, you usually just add singular field, you cannot change existing field usage.
Changing it this way still would be a bc break hidden in a minor then.

Reply:
ok, returned previous code